build-logic-commons/basics/src/main/kotlin/gradlebuild/basics/BuildParams.kt

```diff
@@ -104,6 +106,8 @@ object BuildParams {
     const val FLAKY_TEST = "flakyTests"
     const val INCLUDE_PERFORMANCE_TEST_SCENARIOS = "includePerformanceTestScenarios"
     const val MAX_PARALLEL_FORKS = "maxParallelForks"
+    const val MAX_TEST_DISTRIBUTION_REMOTE_EXECUTORS = "maxTestDistributionRemoteExecutors"
+    const val MAX_TEST_DISTRIBUTION_LOCAL_EXECUTORS = "maxTestDistributionLocalExecutors"
```

Comment:
🤔 You use-case seems to be to disable remote or local executors, I wonder if you would be better off adding flags for those to options instead of general max numbers.